src/styles/index.scss

```diff
@@ -2,2 +1,247 @@
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: black;
+  }
+
+  &__link:hover {
+    color: $blue-accent;
+  }
+
+  .is-active {
+    color: $blue-accent;
+    position: relative;
+  }
+
+  .is-active::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background-color: $blue-accent;
+    border-radius: 2px;
+  }
+}
+
+.product-card {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(1, 408px);
+  gap: 46px 48px;
+
+  grid-auto-flow: row;
+  place-content: center;
+
+  padding: 50px 40px;
+}
+
+.element {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  background: $white-accent;
+  border-radius: 5px;
+  transition: transform 0.3s ease;
+}
+
+.element:hover {
+  transform: scale(1.2);
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 160px 134px;
+  margin: 32px 19px 40px;
+}
+
+.info {
+  margin: 0 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+}
+
+.name {
+  margin: 0;
+  line-height: 18px;
+  margin-bottom: 4px;
+  cursor: pointer;
+}
+
+.name:hover {
+  color: $card-title-color;
+}
+
+.code {
+  margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $secondary;
+  margin-bottom: 6px;
+}
+
+.rating {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  line-height: 16px;
+  margin-bottom: 14px;
+  width: 168px;
+}
+
+.star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  margin-left: 19px;
+}
+
+.price {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 18px;
+  margin-bottom: 16px;
+  width: 168px;
+}
+
+.price-text {
+  color: $secondary;
+  font-weight: 400;
+  margin-right: 85px;
+}
+
+.price-numbers {
+  margin-left: 1px;
+  padding-bottom: 1px;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  font-size: 14px;
+  font-weight: 600;
+  color: $white-accent;
+  background: $blue-accent;
+  border: none;
+  height: 40px;
+  line-height: 37px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: $white-accent;
+  border: 1px solid $blue-accent;
+  color: $blue-accent;
+}
+
+@media (min-width: 488px) {
```

Comment:
Media queries are typically mobile-first, meaning you should start with your smallest styles and then scale up. The current media queries are not mobile-first, consider reordering them.